cmd/cookiemonster/cookie.go

```diff
@@ -6,12 +6,14 @@ import (
 	"github.com/iangcarroll/cookiemonster/pkg/monster"
 )
 
-func handleCookie() {
+func handleCookie(cookieFlag string) {
 	// Attempt to decode this cookie.
-	cookie := monster.NewCookie(*cookieFlag)
+	cookie := monster.NewCookie(cookieFlag)
 
 	if !cookie.Decode() {
-		failureMessage("Sorry, I could not decode this cookie; it's likely not in a supported format.")
+		message := "Sorry, I could not decode this cookie; it's likely not in a supported format."
```

Comment:
Is there a reason this was changed? `failureMessage` should handle this for you.

Reply:
`failureMessage` calls `os.Exit`. So, processing a list isn't possible since it would terminate after 1st try either way.